### PR TITLE
created document

### PR DESCRIPTION
**Pull request by oscarsernarosero**

### CODE REVIEW

File: docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+ | risk score | balance | resultant logic |
+| - | - | - |
+| Implied* | Implied* | 0-24 ->  NO LIMIT |
+| 25 | $500 | 25-49 ->   $500 max |
+| 50 | $250 | 50-74 ->   $250 max |
+| 75 | $50 | 75-100 ->   $50 max |
+
+\* *Note that the first risk segment is implied and has no limit. This first segment is from risk score 0 to the lowest risk score defined in the rule (`riskLevel`). A no-implied-segment rule could be achieved by starting the `riskLevel` from 0.*
+
+***risk scores can be between 0 and 100.***
```
Comment (TJ-Everett) [inline docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md:43]: Risk max score is 99

File: docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+2. The asset handler then sends these values along with the rule Id of the transaction-size-per-period-by-risk-score rule set in the handler, the date of the last transfer of tokens by the account, and the accumulated US Dollar amount transferred during the period to the protocol.
+3. The protocol then proceeds to check if the current transaction is part of an ongoing period, or if it is the first one in a new period.
+    - **If it is a new period**, the protocol resets the accumulated US Dollar amount transferred during current period to just the amount being currently transferred. 
+    - **If it is not a new period**, then the protocol accumulates the amount being currently transferred to the accumulated US Dollar amount transferred during the period. 
+4. The protocol then evaluates the accumulated US Dollar amount transferred during current period against the rule's maximum allowed for the risk segment in which the account is. The protocol reverts the transaction if the accumulated amount exceeds this rule risk-segment's maximum.
```
Comment (TJ-Everett) [inline docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md]: risk segment in which the account is in. *

File: docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    - Function to get the rule Id from the asset handler:
+        ```c
+        function getMaxTxSizePerPeriodByRiskRuleId() external view returns (uint32);
+        ```
+
+## Return Data
+
+This rule returns 1 value:
+
+1. **Accumulated US Dollar Amount Transferred During Period** (int256): the updated value for the total US Dollar amount of tokens during current period.
```
Comment (TJ-Everett) [inline docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md]: uint128

**Review by TJ-Everett — COMMENTED** [review]
Looks good overall Oscar, just a couple things.

File: docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md (hunk 1 above)
Comment (oscarsernarosero) [reply docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md:43]: The actual risk score can be 100. The rules won't be able to be more than 99, which actually makes me realize Im missing that validation. Thank you!

Comment (TJ-Everett) [reply docs/userGuides/rules/TX-SIZE-PER-PERIOD-BY-RISK-SCORE.md:43]: That is totally what I meant! Yes we can add risk scores to 100 but the rule checks for them to be less than 99. I am glad you interpreted my under caffeinated ramblings!

**Review by TJ-Everett — APPROVED** [review]
Looks great sir!

**Review by ShaneDuncan602 — APPROVED** [review]
Well done.

**Pull request merged**